src/backends/torch/torchmodule.cc

```diff
@@ -275,8 +276,24 @@ namespace dd
       return _native->forward(torch_utils::to_tensor_safe(source[0]));
     if (_traced)
       {
-        auto output = _traced->forward(source);
-        source = torch_utils::unwrap_c10_vector(output);
+        if (!forward_method.empty())
+          {
+            for (const auto &method : _traced->get_methods())
+              {
+                if (method.name() == forward_method)
+                  {
+                    _logger->info("found forward method {}", method.name());
+                    auto output = _traced->get_method(forward_method)(
+                        std::move(source));
+                    source = torch_utils::unwrap_c10_vector(output);
+                  }
```

Comment:
Error if not found

Reply:
Actually code for get_method is following:
```c++
  Method get_method(const std::string& name) const {
    if (auto method = find_method(name)) {
      return *method;
    }
    AT_ERROR("Method '", name, "' is not defined.");
  }
```
So we can do
```c++
if (auto method = find_method(name)) {
      _logger->info("found forward method {}", method.name());
      auto output = (*method)(std::move(source));
      source = torch_utils::unwrap_c10_vector(output);
}
else throw MLLibBadParamException("Method " + forward_method + " not found in traced model");
```

Reply:
Done.